staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go

```diff
@@ -154,6 +155,7 @@ func NewCmdExposeService(f cmdutil.Factory, streams genericclioptions.IOStreams)
 	cmd.Flags().String("container-port", "", i18n.T("Synonym for --target-port"))
 	cmd.Flags().MarkDeprecated("container-port", "--container-port will be removed in the future, please use --target-port instead")
 	cmd.Flags().String("target-port", "", i18n.T("Name or number for the port on the container that the service should direct traffic to. Optional."))
+	cmd.Flags().IntVar(&o.NodePort, "node-port", o.NodePort, "Port used to expose the service on each node in a cluster.")
```

Comment:
I would suggest add i18n.T:
```
i18n.T("Name or number..
```

Reply:
Thanks, fixed. I have copied this line from ```create``` command, so I fixed it as well.

https://github.com/kubernetes/kubernetes/blob/3a22c023b9f1a7344ad96343cd68bc7409aee4ba/staging/src/k8s.io/kubectl/pkg/cmd/create/create_service.go#L297

Reply:
@masap different PR ? What's the #?

Reply:
It is in this PR.

Reply:
https://github.com/kubernetes/kubernetes/pull/98244/files#diff-332302c0f142f29cb2425c6e44701022fcaf75515e8ef04181dfa0fc1e5fa8a5